include/fluent-bit/aws/flb_aws_imds.h

```diff
@@ -20,9 +20,9 @@
 #ifndef FLB_AWS_IMDS
 #define FLB_AWS_IMDS
 
-#define FLB_AWS_IMDS_HOST "169.254.169.254"
+#define FLB_AWS_IMDS_HOST "localhost"
```

Comment:
This is just for local testing with my Python server mock. I did want to ensure the implementation still works after using another http client implementation. It's not going to be part of the end goal changes.